src/data/read-through-data-cache.ts

```diff
@@ -132,13 +133,15 @@ export class ReadThroughDataCache implements ContiguousDataSource {
       attributes?.hash,
       attributes?.size,
     );
+
     if (cacheData !== undefined) {
       return {
         hash: attributes?.hash,
         stream: cacheData.stream,
         size: cacheData.size,
-        sourceContentType: dataAttributes?.contentType,
-        verified: dataAttributes?.verified ?? false,
+        sourceContentType: attributes?.contentType,
```

Comment:
@djwhitt I believe the intent was to use `attributes` instead of `dataAttributes` here, right?

Reply:
Yup. Good catch. This looks like a mistake.